torch/_inductor/codegen/triton.py

```diff
@@ -115,6 +115,42 @@ def _print_Abs(self, expr):
         assert len(expr.args) == 1
         return f"tl.abs({self._print(expr.args[0])})"
 
+    def _print_cos(self, expr):
```

Comment:
Not necessary for this PR, but as a follow up we should add these to the `precomputed_replacements` handling in `TritonKernel.indexing` which hoists complex expressions into the wrapper code.